tests/test_xcdat.py

```diff
@@ -1,14 +1,17 @@
 import numpy as np
+import pandas as pd
+import pytest
 import xarray as xr
 
 from tests.fixtures import generate_dataset, lat_bnds
 from xcdat.xcdat import XCDATAccessor
 
 
 class TestXCDATAccessor:
-    # NOTE: We don't have to test this class in-depth because its methods
+    # NOTE: We don't have to perform in-depth testing because its methods
     # are abstracted public methods from other classes, which already has more
     # comprehensive testing.
```

Comment:
```suggestion
    # NOTE: We don't have to perform in-depth testing because the methods
    # are abstracted public methods from other classes, which already has more
    # comprehensive testing.
```